analysis/topeft_run2/analysis_processor.py

```diff
@@ -153,6 +154,16 @@ def process(self, events):
         xsec               = self._samples[dataset]["xsec"]
         sow                = self._samples[dataset]["nSumOfWeights"]
 
+        dt_era = None
+        if year[2] == "2":
```

Comment:
I'm guessing this checks for `202x` but maybe we should think of a better way to check than _just_ the 3rd character.

Reply:
I have implemented here the proposal you made for a similar line on the `topcoffea` side. As a result, I have introduced `is_run2` and `is_run3` boolean flags in place of `dt_era` and adapted the script accordingly

Reply:
I don't see a fix to `if year[2] == "2":`. Can you please update that check to something like you used in `topcoffea`?

Reply:
If we are talking about [these lines](https://github.com/TopEFT/topeft/blob/1b0cfae66cc78ec9c888011a3a3c9b84ee939216/analysis/topeft_run2/analysis_processor.py#L157-L160), the last commit seems to provide such change